styles/main.css

```diff
@@ -5,7 +5,7 @@
 }
 main {
     display: grid;
-    grid-template-columns: 50% 50%;
+    grid-template-columns: 1fr 1fr;

```

Comment:
No marks lost for this, but using % here causes the grid to take more than 100% of the width of its parent because each column takes 50% of its parents and then the column gap adds its space.  Using fr instead causes the columns to take the remaining space AFTER the column gap space has been allocated.